### DISCUSSION TITLE
Hi! I like to have a lazy component skeleton loader

### DISCUSSION THREAD

**Discussion by JenuelDev** [nuxt/nuxt#23111] [Questions]
Hi! I tried making components lazily by prefixing Lazy example `LazyHeaderComponent`. But I like to add skeleton loader,, how do I do that? thanks :)

**Comment by or2e** [comment]
@BroJenuel 
Suspense (#fallback slot)

If your component loads too fast, try setting sleep in setup


Also material on the topic:

https://vuejs.org/guide/components/async.html#loading-and-error-states

https://vuejs.org/guide/built-ins/suspense.html#async-components

https://github.com/nuxt/nuxt/issues/15648
